structs/readme.md

```diff
@@ -26,7 +26,7 @@ Notice the new format string? The `f` is for our `float64` and the `.2` means pr
 ## Write the minimal amount of code for the test to run and check the failing test output
 
 ```go
-func Perimeter(width float64, height float64) (perimeter float64) {
+func Perimeter(width float64, height float64) float64 {
 	return
```

Comment:
This wont compile will it?

Reply:
@quii It will compile right? Otherwise tests would have failed?

Reply:
You cant do naked returns without a named param

The markdown reads

```go
func Perimeter(width float64, height float64) float64 {
	return
}
```
Try it here: https://play.golang.org/p/bHx0k4AdZ0k

Even though the `./build.sh` is successful doesn't mean the markdown is right, as so far i dont know a way to compile the go code inside markdown sadly :(

Reply:
@quii My bad, I overlooked it.
Also I meant I used the same code in one of the go files and it worked fine but it wan't the empty `Parimeter` function. Let me fix this asap.